Dockerfile

```diff
@@ -1,17 +1,29 @@
-FROM node:lts
+FROM node:lts AS builder
 
-# Clone and move into directory
-RUN git clone https://github.com/IntellectualSites/schematic-web.git /app
+# Set working directory
 WORKDIR /app
 
-# Install dependencies
-RUN yarn install
+# Install dependencies without lock file
+COPY package.json /app/package.json
+RUN yarn install --silent
 
-# Build application
-RUN yarn build
+# Get all remaining files
+COPY . /app
+
+# Generate static files
+RUN yarn run generate
+
+FROM nginx:stable-alpine
+
+# Copy dist files (generated) from builder layer into this (lightweight) nginx image
+COPY --from=builder /app/dist /usr/share/nginx/html
 
 # Expose the port
-EXPOSE 3000
+EXPOSE 80
```

Comment:
What is this for?

Reply:
To expose the default nginx port...? Not sure what part about that is unclear :D

Reply:
I am well aware _what_ this does, but I questioned the "why", behind the port change.

Reply:
Because we are not depending on node anymore and the default `yarn serve` command binds on 3000 - now the nginx image is used and therefor the default exposed port has changed. I can change the default port to be changed to 3000, but find 80 more fitting tbh